client/dashboard/app/router.tsx

```diff
@@ -396,6 +397,23 @@ const siteSettingsTransferSiteRoute = createRoute( {
 	)
 );
 
+const siteSettingsWebApplicationFirewallRoute = createRoute( {
+	getParentRoute: () => siteRoute,
+	path: 'settings/web-application-firewall',
+	loader: async ( { params: { siteSlug } } ) => {
+		const site = await queryClient.ensureQueryData( siteBySlugQuery( siteSlug ) );
+		if ( canViewWordPressSettings( site ) ) {
```

Comment:
I think this should be:

```suggestion
		if ( hasHostingFeature( site, HostingFeatures.SECURITY_SETTINGS ) ) {
```

Likewise in `sites/v2/site-settings/router.tsx`

Reply:
Blurgh, this is me forgetting to update a copypasta. 🤦